script/Ciel/Day 3/03_01_CIEL03_3_3C.txt

```diff
@@ -206,37 +229,43 @@ Now that I've calmed down, even I understand that.
 [sha:3a802124435f13b6985d82b8bf0de355b1b8386e]{
 -- Page 259, Offset 5510.
 -- 自分の命を断ったところで何の<償|つぐな>いになる。
-Taking your own life is no atonement for what you've done.
+Taking your own life doesn't serve to atone for your actions.
 }
 [sha:e0d619c2946da399aee08eb522ab2b4c8ead3687]{
 -- Page 259, Offset 5511.
 -- 本当に自分が間違ってしまったと思うなら、安易に逃げる事だけは、するべきではないと思う―――
-If you really think you've made a mistake, you shouldn't just run away...
+If you really think you've done something wrong,
+ I believe that running away from your responsibilities is the last thing you should do―――
 }
 [sha:b3b9a66d0b8c2e833ec246cc619bd3777a71411c]{
 -- Page 260, Offset 5512.
 -- 「罪、ですか。どうも遠野くんの間違いというのは、わたしでは想像もつかないコトみたいですね」
-"Sin? It seems that your mistake is something I can't even imagine."
+"Sins, you say...
+ It seems that whatever you did wrong is something I can't even imagine."
```

Comment:
I kinda prefer mistake.

> It seems that whatever your mistake was is something I can't even imagine.

Reply:
I agree with Yusuke but "was, it must've been something I can't even imagine." or "was, it was something"